samples/mobile_run_test_app/Androidruntests/app/src/main/java/io/multy/android_run_tests/MainActivity.java

```diff
@@ -0,0 +1,33 @@
+package io.multy.android_run_tests;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    // Used to load the 'native-lib' library on application startup.
+    static {
+        System.loadLibrary("multy_core_jnid");
```

Comment:
Release version of the library has a different name, you should load an appropriate version, matching Android application build type.